Lab1.ipynb

```diff
@@ -33,7 +41,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 115,
    "metadata": {},
    "outputs": [
     {
```

Comment:
Explanation for Problem 1B:
- minor typo (calidate --> validate)
- last sentence -- is the probability a value is within 0 and 2 standard deviations above the mean 97.5% or is that the probability integrated from the left tail (-infinity) up to 2 standard deviations above the mean?